inference/core/roboflow_api.py

```diff
@@ -757,7 +757,8 @@ def build_roboflow_api_headers(
     if not ROBOFLOW_API_EXTRA_HEADERS:
         return explicit_headers
     try:
-        extra_headers: dict = json.loads(ROBOFLOW_API_EXTRA_HEADERS)
+        # Avoid unnecessary type annotation here, and don't parse twice
+        extra_headers = json.loads(ROBOFLOW_API_EXTRA_HEADERS)
```

Comment:
```suggestion
        extra_headers: dict = json.loads(ROBOFLOW_API_EXTRA_HEADERS)
```

Reply:
yes, we are currently working on a project that refines the generated code to revert any lines that don't lead to any performance impact.